lib/lighthouse/benefits_documents/service.rb

```diff
@@ -75,7 +75,7 @@ def submit_document(file, file_params, lighthouse_client_id = nil)
       if Flipper.enabled?(:cst_synchronous_evidence_uploads, @user)
         Lighthouse::DocumentUploadSynchronous.upload(user_icn, document_data.to_serializable_hash)
       else
-        Lighthouse::DocumentUpload.perform_async(user_icn, document_data.to_serializable_hash)
+        Lighthouse::DocumentUpload.perform_in(1.minute, user_icn, document_data.to_serializable_hash)
```

Comment:
What's the reason for this change?

Reply:
@jerekshoe We reverted to using `perform_async` by moving the `EvidenceSubmission` record creation responsibility into `DocumentUpload` and out of `Service`